src/cp.js

```diff
@@ -153,6 +171,42 @@ function cpdirSyncRecursive(sourceDir, destDir, currentDepth, opts) {
   } // for files
 } // cpdirSyncRecursive
 
+function major(rdev) {
+  return ((rdev >> 31 >> 1) & 0xfffff000) | ((rdev >> 8) & 0x00000fff);
+}
+function minor(rdev) {
+  return ((rdev >> 12) & 0xffffff00) | (rdev & 0x000000ff);
+}
+
+// attempts to create a dev (character or block) by shelling out to mknod.
+function mknod(destFile, stat) {
+  var created;
+  var failureMessage;
+  var args = ['mknod'];
+  args.push('"' + destFile + '"');
```

Comment:
At first glance this looks wrong. What if `destFile` is `hello world"; rm -rf *; echo "`? Looks like you'll be running:

```sh
$ mknod "hello world"; rm -rf *; echo "" p
```

`JSON.stringify()` is the way we usually perform quote escaping, but you can avoid this entirely if you take my suggestion below.